src/components/DisplayArticle.jsx

```diff
@@ -1,18 +1,46 @@
 import { useParams } from "react-router-dom";
 import { getArticle } from "../utils/api";
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import { increaseVotes } from "../utils/api";
 
 const DisplayArticle = () => {
+  const clickHandler = (e, increment) => {
+    setVotes((currVotes) => {
+      const newVotes = currVotes + increment;
+      return newVotes;
+    });
+    increaseVotes(article_id, increment).then((res) => {});
+  };
+
   const { article_id } = useParams();
   const [selectedArticle, setSelectedArticle] = useState({});
-  getArticle(article_id).then((article) => {
-    setSelectedArticle(article);
-  });
+  const [votes, setVotes] = useState(selectedArticle.votes);
```

Comment:
Here you are setting .votes on undefined! Give this an initial value of 0 and use your setVote function to set the votes once you haves successfully made an API request for your specific article